app.py

```diff
@@ -43,14 +43,12 @@ def get_memory():
     """API endpoint to get memory (RAM and Swap) usage."""
     ram_usage, swap_usage = memory_service.get_memory_usage()
     return jsonify(ram_usage_history=ram_usage, swap_usage_history=swap_usage)
-
-
+    ram_and_swap_usage
```

Comment:
Why adding it here?

Reply:
becouse the ram and  the swap are all in the memory

Reply:
Nope. This is not correct ode. Please fix